localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -366,6 +369,8 @@ void NDTScanMatcher::callback_sensor_points(
     RCLCPP_WARN_STREAM_THROTTLE(this->get_logger(), *this->get_clock(), 1, "No MAP!");
     return;
   }
+  // publish mtt at interpolation
```

Comment:
I think it is difficult to understand what `mtt` is.
How about writing `publish message tracking tag at interpolation` without abbreviation?

Reply:
I avoid using abbreviation as you mentioned at [a673138](https://github.com/autowarefoundation/autoware.universe/pull/2638/commits/a67313824d2de5576a8bc522c384a9940160cc64)